DNN Platform/Website/Install/InstallWizard.aspx

```diff
@@ -666,7 +675,8 @@
                             databaseSecurity: "",
                             databaseUsername: "",
                             databasePassword: "",
-                            databaseRunAsOwner: null
+                            databaseRunAsOwner: null,
+                            dnnImprovementProgram: $('#<%= chkImprovementProgram.ClientID %>')[0].is(":checked") ? "Y" : "N"
```

Comment:
does this work?  `[0]` should return a DOM element (not jQuery-wrapped), which wouldn't have an `is` method…  if you take out the `[0]` it should be fine